kentik_api_library/kentik_api/public/plan.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any, Dict, List, Optional
```

Comment:
Please use @dataclass decorator for all classes in this file (see other modules in .public package for reference)

Reply:
ack